scripts/ngi_pipeline_start.py

```diff
@@ -133,17 +137,18 @@ def __init__(self, *args, **kwargs):
         args.restart_running_jobs = True
 
     # Finally execute corresponding functions
-    ### TODO change to work with multiple flowcells
-    if 'analyze_fc_dir' in args:
-        LOG.info("Starting flowcell analysis in directory {}".format(args.analyze_fc_dir))
-        flowcell.process_demultiplexed_flowcell(args.analyze_fc_dir,
-                                                args.restrict_to_projects,
-                                                args.restrict_to_samples,
-                                                args.restart_failed_jobs,
-                                                args.restart_finished_jobs,
-                                                args.restart_running_jobs,
-                                                quiet=args.quiet,
-                                                manual=True)
+    if 'analyze_fc_dirs' in args:
+        LOG.info('Starting flowcell analysis of flowcell {} '
+                 '{}'.format(inflector.plural("directory", len(args.analyze_fc_dirs)),
```

Comment:
Imagine all the use cases -- it's every noun in the English language! What a module!